lib/Elastica/Search.php

```diff
@@ -22,6 +22,7 @@ class Search
     const OPTION_SIZE = 'size';
     const OPTION_SCROLL = 'scroll';
     const OPTION_SCROLL_ID = 'scroll_id';
+    const OPTION_QUERY_CACHE = 'query_cache';
```

Comment:
Thanks, I've added the test. Btw. the `query_cache` option will probably work only with `search_type=count` but that's expected behavior (https://www.elastic.co/guide/en/elasticsearch/reference/1.6/index-modules-shard-query-cache.html#index-modules-shard-query-cache).